src/pages/vans.jsx

```diff
@@ -1,104 +1,113 @@
+import React from "react";
 import { useSelector } from "react-redux";
 import { useDispatch } from "react-redux";
 import { fetching } from "../features/vanslice";
-import { useLocation, Link, useSearchParams } from "react-router-dom";
 import "../server";
 import { getVans } from "../api";
-import { useLoaderData } from "react-router-dom";
+import { useLoaderData,useLocation, Link, useSearchParams, Await, defer } from "react-router-dom";
 
 
 export function loader(){
-  return getVans()
+  return defer({vans:getVans()})
 }
 
 export default function Vans() {
   const Vans = useLoaderData();
```

Comment:
As it is not a class or a component, start the `Vans` variable with a lowercase character.